src/GitHub.InlineReviews/Properties/DesignTimeResources.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<ResourceDictionary
+        xmlns:cache="clr-namespace:GitHub.UI.Helpers;assembly=GitHub.UI"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <SolidColorBrush x:Key="DiffChangeBackground.Add" Color="Green" />
```

Comment:
If you want any resources to show up in the designer, you can add there here.

You can see below, I'm also adding the 3 `SharedDictionary.xaml` files. At the moment this is only needed for the `Show/AddInlineCommentGlyph.xaml` files (because they're added by `BrushesManager.cs` rather then an explicit `MergedDictionaries` element in each XAML file).